### ISSUE TITLE
Welcome to contributors

### ISSUE THREAD

**Issue by agodemar** [JSBSim-Team/jsbsim-reference-manual#1]
This repo is a proof of concept of how an online JSBSim Manual could possibly evolve.

As of today, we have a MS Word manual edited by Jon Berndt, available as a PDF [here](http://jsbsim.sourceforge.net/JSBSimReferenceManual.pdf).

We've had also an attempt to migrate the content into a LaTeX manual. The source code of this effort is available [here](https://bitbucket.org/jsbsim_manual_team/jsbsim_reference_manual/overview).

This latter solution is stored in a public repository, and being a set of LaTeX files it lends itself to a collaborative approach. Yet, I am aware that not everyone is willing to work with LaTeX.

I recently discovered an interesting feature on GitHub called [GitHub Pages](https://pages.github.com).

I did an experiment with this free service based on a technology called Jekyll (sitting on top of Ruby), and the result is this website: [FlightMechanics4Pilots](https://agodemar.github.io/FlightMechanics4Pilots)

The website sources live are [here](https://github.com/agodemar/FlightMechanics4Pilots/tree/master/docs). It showcases some features that might be adopted in an online user manual of JSBSim:
- all sources are text files in Markdown format, very useful for
  collaborative and concurrent development
- uses LaTeX formulas via the MathJax Javascript engine
- the images are in PNG or even SVG format.

To develop the website locally on my PCs, I use Ruby+Jekyll from an Ubuntu bash shell on Windows 10. This is possible thanks to the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10).

Now, you know that we also have a GitHub repository [here](https://github.com/JSBSim-Team/jsbsim) that mirrors the original one on SourceForge.

I think it could be a good idea to take advantage of GitHub to build up an online manual. I can do the initial work to setup the file structure and prepare the skeleton of the draft website.

At that point volunteers can contribute to the content with pull requests.

**Comment by seanmcleod** [comment]
Hi Agostino

I had actually pinged the JSBSim mailing list a couple of times now asking about the source of the PDF manual since I wanted to contribute some updates in terms of the new axes support we added recently and the new template functions.

So I'm interested in contributing. 

Are you planning on starting by converting the current Word/Latex text into a GitHub Pages version, or are you planning on starting from scratch?

Cheers

**Comment by andgi** [comment]
Hi Agostino,

Do you have a license in mind for this new manual?
The two previous ones seems to be closed/proprietary but that might not be the best option? Not sure what might be, though.

I have a short section on the buoyant forces that I contributed to the old manual which I could contribute here as well and I have some other HowTo material on the FlightGear wiki but those are GPLv2 (though I suppose I could contribute them on other terms in the cases where I'm the only author).

Cheers

**Comment by agodemar** [comment]
@seanmcleod 
I'll need to set up in place a the files that reproduce the structure of the original Word document by Jon. The menu of this website is directed by this [contentlist.yml](https://github.com/JSBSim-Team/jsbsim-user-manual/blob/master/docs/_data/contentlist.yml) file. The files referenced to in the menu are located in the path [/docs/_mypages/](https://github.com/JSBSim-Team/jsbsim-user-manual/tree/master/docs/_mypages). These files are in Markdown format and represent the content of the website.

Then, I'll start by converting the current Word/Latex text into a GitHub Pages version. But here we can discuss possible adjustments and extensions.

**Comment by agodemar** [comment]
@andgi 
I do not have in mind a specific license. I think it should be as simple, easy and liberal as possible. If you have an idea, please just let us know.

**Comment by agodemar** [comment]
Of course, everyone is welcome to add content and help in the migration. If you are in the group of collaborators of this repo you can do a push yourself. Those who are not can do a pull request.
All you have to di is exploring the folder [docs/_mypages](https://github.com/JSBSim-Team/jsbsim-user-manual/tree/master/docs/_mypages) of this repo.
On request, for those who are not familiar with GitHub Pages, Ruby, and Jekyll, I can privately help with setup of all local tools on Windows.

**Comment by seanmcleod** [comment]
Agostino did you use the following instructions for installing Ruby and Jekyll using the Windows Bash feature?

https://jekyllrb.com/docs/windows/

**Comment by agodemar** [comment]
@seanmcleod Yes. Do not install Ruby for Windows. Use Ruby from the Ubuntu shell on Windows 10.

**Comment by bcoconni** [comment]
@agodemar
What about the Doxygen documentation ? At the moment, it is copied manually to http://jsbsim.sourceforge.net but we might consider including it to this project as well ?

**Comment by agodemar** [comment]
@bcoconni Having the Doxygen documentation in this project would be nice.
